src/PTFAnalysis.cpp

```diff
@@ -565,13 +712,13 @@ PTFAnalysis::PTFAnalysis( TFile* outfile, PTF::Wrapper & wrapper, double errorba
       if(0)std::cout << "Check save waveform: " << save_waveforms << " " << savewf_count
 << " " << savenowf_count << " " << curscanpoint.x() << std::endl; 
       // check if we should clone waveform histograms
-      if ( save_waveforms && savewf_count<500 && savenowf_count<500 ){
+      if ( save_waveforms && savewf_count<1000 && savenowf_count<1000 ){
```

Comment:
add configurable parameter?

Reply:
I reverted this change. 

Having it be a configurable parameter makes sense; let's do separate PR for that.